runtime/src/constants.rs

```diff
@@ -7,7 +7,7 @@ pub mod currency {
 	pub const MILLICENTS: Balance = CENTS / 1_000; // 1_000_000
 
 	pub const fn deposit(items: u32, bytes: u32) -> Balance {
-		items as Balance * 15 * CENTS + (bytes as Balance) * 6 * CENTS
+		items as Balance * 2_000 * CENTS + (bytes as Balance) * 100 * MILLICENTS
```

Comment:
This one from Kusama:
https://github.com/paritytech/polkadot/blob/aeea9b7bd81919e014f7621f6c4b2eb9709d918f/runtime/kusama/src/constants.rs#L27